program-test/src/lib.rs

```diff
@@ -1002,6 +1002,7 @@ impl ProgramTestBanksClientExt for BanksClient {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
Just glancing here, it seems like we should maybe add this
```rust
self.1.join()
```
to this function:
https://github.com/anza-xyz/agave/blob/fadfa6140c1a97834a50274f3092538e910f14c7/program-test/src/lib.rs#L1007-L1011

I'm fine adding `dead code` for this one

Reply:
I think we can just abort it 🤔 [acf3482](https://github.com/anza-xyz/agave/pull/1246/commits/acf3482feb9263c954d8b985cac5df8188486a78)

Reply:
I stole your idea here: https://github.com/anza-xyz/agave/pull/1246/commits/c60031b8fcb5adf00f130429ad64720c64578a8f 🥷